meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -211,6 +211,10 @@ PREFERRED_VERSION_optee-client_imx7ulpea-ucom = "3.6.0+git"
 PREFERRED_VERSION_optee-examples_imx7ulpea-ucom = "3.6.0"
 PREFERRED_VERSION_optee-test_imx7ulpea-ucom = "3.6.0"
 OPTEE_BINARY_imx7ulpea-ucom = "tee-pager.bin"
+UBOOT_MACHINE_imx7ulpea-ucom = "mx7ulp_com_defconfig"
+UBOOT_CONFIG_imx7ulpea-ucom = ""
+BOOTSCR_LOAD_ADDR_imx7ulpea-ucom = "0x60800000"
```

Comment:
This change also requires this change https://github.com/foundriesio/meta-lmp/pull/369/files#diff-4ad7fe4e4bf5a699191b63e8be4ede49009acad31383b25c8f47b6128c0c2dd7R53,

I'd suggest to keep these two here:
```
UBOOT_MACHINE_imx7ulpea-ucom = "mx7ulp_com_defconfig"
UBOOT_CONFIG_imx7ulpea-ucom = ""
```
and just cherry-pick my last two patches from https://github.com/foundriesio/meta-lmp/pull/369/commits